src/load/azext_load/data_plane/load_test_run/custom.py

```diff
@@ -236,15 +250,28 @@ def download_test_run_files(
     if test_run_input:
         _download_input_file(test_run_input_artifacts, test_run_id, path)
 
+    is_high_scale_test_run = _is_high_scale_test_run(test_run_data)
+    high_scale_test_run_message = ""
     if test_run_log:
-        _download_logs_file(test_run_output_artifacts, test_run_id, path)
+        if is_high_scale_test_run:
+            high_scale_test_run_message += f"High scale test run {test_run_id} log file " \
+                "is not available for download. "
```

Comment:
@saloniagrawal1997 to review

Reply:
Log file for high-scale test {test_run_id} is not available for download. Use the 'get-artifacts-url' command to fetch the SAS URL and access the file.